Library/Formula/osquery.rb

```diff
@@ -52,9 +53,9 @@ def install
   test do
     require 'open3'
     Open3.popen3("#{bin}/osqueryi") do |stdin, stdout, _|
-      stdin.write(".mode line\nSELECT major FROM osx_version;")
+      stdin.write(".mode line\nSELECT version FROM osquery_info;")
       stdin.close
-      assert_equal "major = 10\n", stdout.read
+      assert_equal "version = #{version}\n", stdout.read
```

Comment:
It is broadly up to you, yes. On occasion we've had to deviate slightly when e.g. a character is introduced which causes us issues because of how we prefix things on disk. We also prefer not to use version lookups in tests because for most software (albeit not `osquery`) it results in a bad test and I'd rather not confuse contributors by having to justify why it's OK for `osquery` and not for their software.

Please make the change that @tdsmith requested, thanks.